### DISCUSSION TITLE
Push HW_03 to Tue (10/31)?

### DISCUSSION THREAD

**Discussion by cooperrc** [cooperrc/computational-mechanics#212] [Polls]
Greetings, 
Yesterday in class I had to run out, but there seemed to be a number of people in the process of working through HW_03 that needed an extra class session to complete the work. 

Can we push HW_03 due date to Tue (10/31) and Project_03 to Tue (11/7)? It will give us one less class for Module_04 work, but I think we can catch up.

**Comment by avanickerson** [comment]
Dear Professor Cooper, I realize your preferred method of contact is GitHub. However, I have sent you an email and was wondering if you could take a look at it when you get the chance. Thank you :)
